features/s3/postObject.feature

```diff
@@ -0,0 +1,9 @@
+@s3 @integ
+Feature: POST object uploads
+
+    Scenario: Upload and object via POST
```

Comment:
This one scenario is testing both V2 and V4 signature POST uploads. They should be split into separate scenarios.